### ISSUE TITLE
Doesn't work properly if Spicetify is installed

### ISSUE THREAD

**Issue by NguyenASang** [mrpond/BlockTheSpot#545]
- [x] You are using the Spotify `last tested version` mentioned in the [README](https://github.com/mrpond/BlockTheSpot/blob/master/README.md).
 - [x] You are using the latest [BlockTheSpot](https://www.github.com/mrpond/BlockTheSpot/releases) from github.
 - [x] You uninstalled Spotify and just ran the patch
 - [x] You read through the [known issues](https://github.com/mrpond/BlockTheSpot/blob/master/README.md#known-issues) section in the README.
 - [x] You read through the [additional notes](https://github.com/mrpond/BlockTheSpot/blob/master/README.md#additional-notes) section in the README.
 - [x] You enabled logging by editing config.ini `set log = 1`.
 - [x] Make sure whether an issue related yours is already open or not.
 - [x] Make sure you have completed everything in the above checklist.

## Screenshots:
![Screenshot 2024-06-17 092935](https://github.com/mrpond/BlockTheSpot/assets/87893636/188343c4-4249-42d3-b0db-3fb19185bdbd)

![Screenshot 2024-06-17 092854](https://github.com/mrpond/BlockTheSpot/assets/87893636/e04f310d-9bfe-430b-a262-c0904f2710f7)

## Log:
Although the log shows the script working fine, I still upload here just in case: [blockthespot.log](https://github.com/user-attachments/files/15860943/blockthespot.log)

**Comment by Nuzair46** [comment]
You should install it after spicetify install

**Comment by Nuzair46** [comment]
can you:
- install Spicetify
- Install BlockTheSpot with manual step.

**Comment by NguyenASang** [comment]
still same issue

**Comment by NguyenASang** [comment]
@Nuzair46 I found the reason why.

- When we run `Spicetify apply` or `Spicetify backup apply` command, Spicetify will move `%APPDATA%\Spotify\Apps\*.spa` to `%APPDATA%\spicetify\Backup` folder and leave a extracted version.

- Therefore, when we run our batch, it can't file `*.spa`, so it can't patch it.

### How to fix ?
- Pack `login` and `xpui` folders to zip then rename packed folders' extension to `.spa`.
- Rerun our batch and everything works fine.

I think we should implement a way to patch inside the `login` and `xpui` folders as well (or pack it from those folder if packed version are not found).

**Comment by Nuzair46** [comment]
Yes, I know that, but manual installation doesn't modify xpui file directly.

**Comment by NguyenASang** [comment]
What's the different between manual installation and automated installation ?
I took a look at `install.ps1`, it does exactly what manual steps do.

**Comment by DerDemystifier** [comment]
Spicetify has its own AdBlock extension. Why'd you use BlockTheSpot to begin with!

**Comment by Constrat** [comment]
@NguyenASang Sorry for the bother, but I wanted to ask you if it was still working, I tried various methods, automatic script for both, separate scripts, one before the other and viceversa, manual both, but I can't seem to be able to install BTS over Spicetify. 
Spicetify will always "prevail" making the `Explore Premium` appear, even if I patch Spotify with BTS after having finished with Spicetify.

I also tried your method but I honestly don't think I did it correctly as nothing "new" happened.
